qualimap/qualimap.cwl

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+
+# Metadata
+label: qualimap-qc
+doc: This is qualimap CWL tool definition http://qualimap.bioinfo.cipf.es/
+
+hints:
+  ResourceRequirement:
+    ramMin: 4000
+    coresMin: 1
+  SoftwareRequirement:
+    packages:
+      qualimap:
+        version: [ "2.2.2d" ]
+        specs:
+          - https://identifiers.org/rrid/RRID:SCR_001209
+          - https://bio.tools/qualimap
+  DockerRequirement:
+    dockerPull: 'quay.io/biocontainers/qualimap:2.2.2d--1'
+requirements:
+  - class: ShellCommandRequirement
+
+# Base command
+baseCommand: [ "qualimap"]
+
+# Perform RNA-seq QC analysis on paired-end data http://qualimap.bioinfo.cipf.es/doc_html/command_line.html
+# Desired java memory size can be set using --java-mem-size option
+arguments:
+  - valueFrom: |-
+      rnaseq --paired --java-mem-size="$(inputs.javamem)"
+    shellQuote: false
+
+inputs:
+  javamem:
+    type: string
+    doc: |
+      Set desired Java heap memory size
```

Comment:
Shouldn't this be optional? Or at least have a default value?

Reply:
Added a default value in [ec7857b](https://github.com/common-workflow-library/bio-cwl-tools/pull/78/commits/ec7857bd428a750c58f722eef84e825687eb1f3d)